code/game/machinery/portable_turret.dm

```diff
@@ -453,6 +453,7 @@ var/list/turret_icons
 		enabled = 0 //turns off the turret temporarily
 		sleep(60) //6 seconds for the traitor to gtfo of the area before the turret decides to ruin his shit
 		enabled = 1 //turns it back on. The cover popUp() popDown() are automatically called in process(), no need to define it here
+		makeSpeedProcess() //GOTTA GO FAST

```

Comment:
When `die()` is called (when the turret dips below 0 health), should I call `makeSlowProcess()` on it?